Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -103,6 +108,26 @@
         return makeVector(result.get());
     }
 
+    void webCryptoMasterKeyAsync(CompletionHandler<void(std::optional<Vector<uint8_t>>&&)>&& completionHandler) final
```

Comment:
This should be just a part of webCryptoMasterKey.  If we want to keep supporting the synchronous selector, then call the synchronous selector and call the completion handler immediately with its result.

Reply:
UIDelegate::UIClient::createNewPage is an example of a single C++ call that calls one of several selectors, one async and one sync.